runhouse/rns/resource.py

```diff
@@ -53,7 +53,7 @@ def _resource_string_for_subconfig(self, resource):
                 # Calls save internally and puts the resource in the current folder
                 resource.name = rns_client.resolve_rns_path(resource.rns_address[1:])
             # TODO [DG] not sure if we should save here
-            resource.save()
+            # resource.save()
```

Comment:
@dongreenberg I think we don't need this now that we enforce calling `save()` more explicitly on resources
wdyt?